acceptance/current/.acceptance/acceptance-cookbook/recipes/destroy.rb

```diff
@@ -0,0 +1,3 @@
+execute 'kitchen destroy' do
+  cwd File.join(File.dirname(__FILE__), "../../../../../../../..")
```

Comment:
I think it's larger than just the kitchen execution (and any associated chef resources). The `chef-client` run that the chef-acceptance CLI command (or eventual resource) executes should have a `dns.json` with useful node attribute data. This is the pattern that delivery-cli follows with great success:
https://github.com/chef/delivery-cli/blob/7acaafed2291e644eed97fdda07ac634ad5f8d6f/src/delivery/job/workspace.rs#L457-L480